packages/manager/apps/pci-gateway/src/components/list/PrivateNetworkButton.component.tsx

```diff
@@ -0,0 +1,47 @@
+import { useEffect, useState } from 'react';
+import {
+  ODS_THEME_COLOR_INTENT,
+  ODS_THEME_TYPOGRAPHY_SIZE,
+} from '@ovhcloud/ods-common-theming';
+import {
+  ODS_BUTTON_SIZE,
+  ODS_BUTTON_VARIANT,
+  ODS_TEXT_LEVEL,
+} from '@ovhcloud/ods-components';
+import { OsdsButton, OsdsText } from '@ovhcloud/ods-components/react';
+import { useTranslation } from 'react-i18next';
+import { useNavigation } from '@ovh-ux/manager-react-shell-client';
+
+export const PrivateNetworkButton = ({ projectId }: { projectId: string }) => {
+  const { t } = useTranslation();
+  const [url, setUrl] = useState('');
+  const nav = useNavigation();
+
+  useEffect(() => {
+    nav
+      .getURL(
+        'public-cloud',
+        `#/pci/projects/${projectId}/private-networks`,
+        {},
+      )
+      .then(setUrl);
+  }, []);
```

Comment:
maybe add projectId to the change trigger, this way you're sure that the url is accurate in all circumstances